fdbclient/admin_client.go

```diff
@@ -459,7 +459,15 @@ func (client *cliAdminClient) KillProcesses(addresses []fdbv1beta2.ProcessAddres
 	if len(addresses) == 0 {
 		return nil
 	}
-	_, err := client.runCommand(cliCommand{command: fmt.Sprintf(
+
+	version, err := fdbv1beta2.ParseFdbVersion(client.Cluster.Status.RunningVersion)
+	if err == nil && version.IsAtLeast(fdbv1beta2.Versions.SupportsBindingKill) {
```

Comment:
In addition we have to add a new setting "UseManagementAPI" to guard those settings with a setting or we make use of a feature flag.